Sources/Crypto/Keys/EC/X25519Keys.swift

```diff
@@ -54,6 +54,14 @@ extension Curve25519 {
             private func withUnsafeBytes<R>(_ body: (UnsafeRawBufferPointer) throws -> R) rethrows -> R {
                 return try self.baseKey.keyBytes.withUnsafeBytes(body)
             }
+			
+			public static func ==(lhs: Self, rhs: Self) -> Bool {
```

Comment:
Part of #173 but I was unable to create a diff between just this PR and #173 since my branch is in a fork, otherwise I would have created this PR with the branch `curve25519_publickeys_equatable` as the target branch, so that we could have seen only the relevant diff.